tests/test_main.py

```diff
@@ -397,6 +397,49 @@ class Model(BaseModel):
     ]
 
 
+def test_const_list():
+    class SubModel(BaseModel):
+        b: int
+
+    class Model(BaseModel):
+        a: List[SubModel] = Schema([{'b': 1}, {'b': 2}, {'b': 3}], const=True)
+
+    m = Model()
+    assert m.a == [{'b': 1}, {'b': 2}, {'b': 3}]
```

Comment:
can you add a test for `Model(a=[SubModel(b=1), SubModel(b=2), SubModel(b=3)])`

Reply:
done